docs/classic-ui/static-resources.md

```diff
@@ -52,15 +56,15 @@ You can register a CSS resource in the same way.
   </registry>
 ```
 
-Registering a JavaScript file and a CSS file in the same bundle is also possible.
+You can also register a JavaScript file and a CSS file in the same bundle.
```

Comment:
"It is also possible to register a JavaScript file and a CSS file in the same bundle." If you don't want to use "you".
```suggestion
It is also possible to include both a JavaScript file and a CSS file in the same bundle.
```

Reply:
I want to address the reader with the active voice of "you can", not the passive voice of "it is possible". The former tone is more encouraging and friendly to the reader. It's a thing we adopted from the [Microsoft Writing Style Guide](https://learn.microsoft.com/en-us/style-guide/grammar/verbs#active-and-passive-voice), as noted in https://6.docs.plone.org/contributing/documentation/authors.html#plone-documentation-styleguide.